src/src/tx_main.cpp

```diff
@@ -634,18 +647,21 @@ bool ICACHE_RAM_ATTR RXdoneISR(SX12xxDriverCommon::rx_status const status)
 
 void ICACHE_RAM_ATTR TXdoneISR()
 {
-  HandleFHSS();
-  HandlePrepareForTLM();
-#if defined(Regulatory_Domain_EU_CE_2400)
-  if (TelemetryRcvPhase != ttrpPreReceiveGap)
+  if (connectionState != tentative)
```

Comment:
Isn't this going to be a problem when the connection state goes to tentative state because theres a connection being established with the RX?
I think it should be a new state rather that co-opting the current tentative RF connection state.

Reply:
The TX doesn't use `tentative` state at all prior to this PR, so there's no chance if it hitting any other TX code with a tentative check. It seemed redundant to have a state that's just used on the RX between disconnected and connected on the RX and one that's just used on the TX between CRSF disconnected (noCrossfire) and CRSF connected (disconnected). I could add a new state `tentativeCrsf` and rename the RX one to `tentativeRx` if you think it is confusing though?

The only thing we have to really worry about is that the "active" TX code assumes only two states: connected, disconnected. I checked through all the code and I *think* we're safe in that area.

Reply:
I tested it and it worked as expected, removing and replacing the module in the bay while connected.
I think two states might make it clearer and the `tentativeCrsf` (or perhaps `awaitingModelId`) should be in the MODE_STATES section.

Reply:
Yup, makes sense to me, changed to awaitingModelId